edx_sg_block/sga.py

```diff
@@ -679,3 +681,10 @@ def require(assertion):
     """
     if not assertion:
         raise PermissionDenied
+
+
+def check_user_access(user, course):
+    for level in ['instructor', 'staff']:
+        if has_access(user, level, course):
+            return True
+    return False
```

Comment:
Nit: `return any(has_access(user, level, course) for level in ['instructor', 'staff'])`

Reply:
👍

Reply:
DRY baby DRY, there should be a method semantically same as this one. You should search for `is_staff_or_instructor_on_course` and reuse that method.